Makefile

```diff
@@ -6,7 +6,7 @@ all: deps lint man package install
 
 # Install dependent pip packages, needed to lint or build
 deps:
-	$(PYTHON) -m pip install black isort pylint mypy build twine pytest pytest-mock
+	$(PYTHON) -m pip install .[dev]
 	$(PYTHON) -m pip install .
```

Comment:
Oh sorry `.[dev]` includes installing `.`, you shouldn't need both

Reply:
Pip has no way afaik to just install dependencies of a package, which is incredibly silly